tests/test_status_codes.py

```diff
@@ -0,0 +1,25 @@
+import http
+import sys
+from typing import Tuple
+
+import pytest
+
+from falcon import status_codes
+
+
+class TestStatusCodes:
+    @pytest.mark.skipif(
+        sys.version_info < (3, 13), reason='Outdated http statues definitions'
+    )
+    def test_statuses_are_in_compliance_with_http_with_rfc(self):
```

Comment:
I don't think we should call this test as "compliance with RFC", as CPython is not in strict compliance either.